docs/user/icl/local-vc-authentication.inc

```diff
@@ -0,0 +1,61 @@
+Cloning a repository
+^^^^^^^^^^^^^^^^^^^^
+
+You can use Sourcetree, Git from the terminal, or any client you like to clone your Git repository.
+These instructions show you how to clone your repository using Git from the terminal.
+
+From the exercise view, click the code button to display the Clone dialog.
+Copy the clone URL (either the SSH format or the HTTPS, with or without token).
+If you are using the SSH protocol, ensure your public key is stored in your Artemis account settings and loaded on the local system to which you are cloning.
+From a terminal window, change to the local directory where you want to clone your repository.
+
+Paste the command you copied from Bitbucket, for example:
+
+Clone over HTTPS:
+
+.. code-block:: bash
+
+   git clone https://username@artemis.cit.tum.de/course/documentation-tests.git
+
+
+Clone over HTTPS with access token:
+
+.. code-block:: bash
+
+   git clone https://username:<ACCESS-TOKEN>@artemis.cit.tum.de/course/documentation-tests.git
```

Comment:
I feel like the `<Templating>` is done a bit inconsistently - "username" is not templated, "ACCESS_TOKEN" is. Please align those to each other